lib/mongoose_push.ex

```diff
@@ -63,7 +63,7 @@ defmodule MongoosePush do
           :ok | {:error, Service.error()} | {:error, MongoosePush.error()}
   def push(device_id, %{:service => service} = request) do
     mode = Map.get(request, :mode, :prod)
-    module = MongoosePush.Application.services()[service]
+    module = Application.get_env(:mongoose_push, MongoosePush.Service) |> Keyword.get(service)
```

Comment:
All configuration should be handled by `MongoosePush.Application` module - if we even would change method of configuration, it should be localized to only that module. Let's modify original `MongoosePush.Application.services()` instead.

Reply:
I do understand that this will affect more then just `push` function on startup. But for this reason Mox has: https://hexdocs.pm/mox/Mox.html#stub_with/2 - to provide default implementation for functions that we do not plan on mocking.